usaspending_api/search/delta_models/subaward_search.py

```diff
@@ -563,7 +566,16 @@
         CONCAT(rl_state_fips.fips, rl_county_fips.county_numeric) AS legal_entity_county_fips,
         CONCAT(pop_state_fips.fips, pop_county_fips.county_numeric) AS place_of_perform_county_fips,
         UPPER(COALESCE(fpds.place_of_perform_county_na, fabs.place_of_perform_county_na)) AS pop_county_name,
-        tas.program_activities
+        tas.program_activities,
+        CASE
```

Comment:
Since you are doing a LEFT JOIN down below doesn't this CASE statement have the possibility to return `-R` or `-C` without a Recipient Hash?

Reply:
For example, what if a Subaward either doesn't contain a UEI / DUNS or it does but it doesn't match a value in the Recipient Profile table.